frontend/app/layout.tsx

```diff
@@ -105,7 +106,10 @@ export default function RootLayout({
                strategy="beforeInteractive"
             />
          </head>
-         <body>{children}</body>
+         <body>
+            {children}
+            <SpeedInsights sampleRate={0.2} />
```

Comment:
20% of requests seems like a lot, but it looks like that's what we were doing before